tffm/core.py

```diff
@@ -94,25 +94,29 @@ class TFFMCore():
     Steffen Rendle, Factorization Machines
         http://www.csie.ntu.edu.tw/~b97053/paper/Rendle2010FM.pdf
     """
-    def __init__(self, order=2, rank=2, input_type='dense', loss_function=utils.loss_logistic, 
-                optimizer=tf.train.AdamOptimizer(learning_rate=0.01), reg=0, init_std=0.01, 
-                use_diag=False, reweight_reg=False, seed=None, use_weights=False):
+    def __init__(
+        self, n_features, order=2, rank=2, input_type='sparse',
```

Comment:
Update comments to include `model_type` and `n_features` as parameters and remove `n_features` from attributes